### ISSUE TITLE
Sometimes it happens that the dock is hidden, and there's no way to get up

### ISSUE THREAD

**Issue by JenaPlinsky** [psifidotos/Latte-Dock#103]
Sometimes it happens that the dock is hidden, and there's no way to get up? ...one can see the icons that flicker and after a bit of random attempts reappears? ...an I help you in some way?

**Comment by psifidotos** [comment]
It would be good to know how to reproduce it

**Comment by JenaPlinsky** [comment]
```
Ivan@UltrabookSamsung:~> latte-dock --debug
qt5ct: using qt5ct plugin
[debug 20:15:55.311311] - package is valid true
[debug 20:15:55.311311] - Latte::DockCorona the package QJsonObject({"KPlugin":{"Authors":[{"Email":"mvourlakos@gmail.com, audoban@openmailbox.org","Name":"Michail Vourlakos, Smith Ar"}],"Description":"Shell provided for the Latte Dock","Id":"org.kde.latte.shell","License":"GPLv3+","Name":"Latte Shell","ServiceTypes":["Plasma/Shell"],"Version":"0.5.91","Website":"https://github.com/psifidotos/Latte-Dock"},"Keywords":"shell","X-KDE-ParentApp":"latte-dock"}) is valid!
qt5ct: using qt5ct plugin
qt5ct: using qt5ct plugin
Config timer connecting...
Setting the name of  0x150fc10  to  "org.kde.ActivityManager.Resources.Scoring"
Creating directory:  "/home/Ivan/.local/share/kactivitymanagerd/resources/"
KActivities: Database connection:  "kactivities_db_resources_140133385209728_readwrite" 
    query_only:          QVariant(qlonglong, 0) 
    journal_mode:        QVariant(QString, "wal") 
    wal_autocheckpoint:  QVariant(qlonglong, 100) 
    synchronous:         QVariant(qlonglong, 1)
Setting the name of  0x15d4d10  to  "org.kde.ActivityManager.RunApplication"
Setting the name of  0x15b8f00  to  "org.kde.ActivityManager.ActivityTemplates"
Service started, version: 7.0.0
Cleaning up...
[debug 20:15:56.667667] - Adding dock for container...
[debug 20:15:56.936936] - SOURCE: QUrl("/usr/share/plasma/shells/org.kde.latte.shell/contents/views/Panel.qml")
[warning 20:15:57.632632] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:15:57.632632] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:15:57.633633] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:15:57.633633] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:15:57.633633] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:15:57.633633] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.2222] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.2222] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.2222] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.2222] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.2323] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:15:58.2323] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:15:58.178178] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
qt5ct: using qt5ct plugin
[warning 20:15:58.708708] - Trying to use rootObject before initialization is completed, whilst using setInitializationDelayed. Forcing completion
[debug 20:15:58.708708] - Reading places from configuration: [{ "providerId": "yrno", "placeIdentifier": "Germany/North_Rhine-Westphalia/Bonn", "placeAlias":"Bonn" }, { "providerId": "yrno", "placeIdentifier": "USA/New_York/New_York", "placeAlias": "New York" }]
[debug 20:15:58.709709] - Reading places from configuration: [{ "providerId": "yrno", "placeIdentifier": "Germany/North_Rhine-Westphalia/Bonn", "placeAlias":"Bonn" }, { "providerId": "yrno", "placeIdentifier": "USA/New_York/New_York", "placeAlias": "New York" }]
[debug 20:15:58.710710] - cacheBackend failed to initialize
[debug 20:15:58.710710] - Reading places from configuration: [{ "providerId": "yrno", "placeIdentifier": "Germany/North_Rhine-Westphalia/Bonn", "placeAlias":"Bonn" }, { "providerId": "yrno", "placeIdentifier": "USA/New_York/New_York", "placeAlias": "New York" }]
[warning 20:15:58.784784] - Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
[warning 20:15:59.149149] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[warning 20:15:59.149149] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[warning 20:15:59.149149] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[warning 20:15:59.149149] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[debug 20:15:59.645645] - ----------- Latte Plasmoid Signal: Activities number was changed ---------
[debug 20:15:59.645645] - 583e7863-7562-40ad-aed6-37dc244d3be9
[debug 20:15:59.645645] - d03c5234-4009-465a-9da3-87bc853a4c3e
[debug 20:15:59.645645] - 077609ba-74dc-4150-8d92-74a46b23a404
[debug 20:15:59.645645] - 75297778-6994-417b-b5e6-ba0cc0934057
[debug 20:15:59.645645] - c7b541ec-c223-4660-9237-aaa69c2ddb10
[debug 20:15:59.645645] - d6472788-db57-4a05-96b8-299c4853e819
[debug 20:15:59.645645] - 3c96aa8e-b757-4a5e-ba20-90dd33f7a969
[debug 20:15:59.645645] - adc86a77-072c-4231-99e7-108225d83594
[debug 20:15:59.645645] - e95c9977-9b8f-4ee4-95b3-b7038bf5c630
[debug 20:15:59.645645] - afaa6c8d-ff8b-49ab-b357-b2d2b48a4009
[debug 20:15:59.645645] - 32ce0942-edce-41b4-b5a5-fab3a366926b
[debug 20:15:59.646646] - ----------- Latte Plasmoid Signal End ---------
[warning 20:15:59.704704] - Trying to use rootObject before initialization is completed, whilst using setInitializationDelayed. Forcing completion
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:15:59.762762] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:15:59.923923] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.923923] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.923923] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.924924] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:15:59.924924] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:15:59.924924] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:16:00.7979] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:16:00.218218] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:16:00.242242] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/Button.qml:99: TypeError: Cannot read property of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:353: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:352: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:351: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/ScrollView.qml:350: TypeError: Cannot read property 'padding' of null
[warning 20:16:00.375375] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:905: TypeError: Cannot read property '__selectionHandle' of null
[warning 20:16:00.376376] - file:///usr/lib64/qt5/qml/QtQuick/Controls/TextArea.qml:941: TypeError: Cannot read property '__cursorHandle' of null
[warning 20:16:00.593593] - Both point size and pixel size set. Using pixel size.
[warning 20:16:00.852852] - libpng warning: iCCP: known incorrect sRGB profile
[debug 20:16:01.739739] - detected kglobalaccel restarting, re-registering all shortcut keys
[debug 20:16:02.756756] - klauncher not running... launching kdeinit
kdeinit5: Shutting down running client.
qt5ct: using qt5ct plugin
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
[debug 20:16:04.911911] - Step 3 - found:56
libva info: VA-API version 0.39.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib64/dri/i965_drv_video.so
libva info: Found init function __vaDriverInit_0_39
libva info: va_openDriver() returns 0
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/akregator.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/amarok.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/amarok.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/blogilo.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/disable_kmix.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/fonts_akregator.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/fonts_global.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/fonts_kate.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/gtkbreeze5.5.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/importwizard.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kaddressbook.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kalarm.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/katomic-levelset.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/katomic-levelset.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kcalcrc.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/kcalcrc.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kde4breeze.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kget.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/kget.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kgthemeprovider-migration.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/kgthemeprovider-migration.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kleopatra.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/klickety.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/klickety.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kmail.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/knavalbattle.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/knavalbattle.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/knotes.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kontact.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/konversation.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/konversation.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/korganizer.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/krdb_libpathwipe.upd' for new updates
"Missing \"Version=5\", file '/usr/share/kconf_update/krdb_libpathwipe.upd' will be skipped."
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/krunnerplugins.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/kscreenlocker.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/messageviewer.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/pimsettingexporter.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/sieveeditor.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/storageservicemanager.upd' for new updates
2017-01-31T20:16:07 Checking update-file '/usr/share/kconf_update/webengineurlinterceptoradblock.upd' for new updates
[debug 20:16:08.921921] - Step 3 - found:48
[debug 20:16:09.112112] - Step 3 - found:40
[debug 20:16:09.552552] - Something went wrong, received HTTP status code 403
```
here I just upgraded Latte from master branch and than started in debug mode
I am in this condition (casually :) ) and I am waiting for your suggestions, if you want
- Hidden and flickering dock
- I can see only very partial icons
- The triggering event might be the automatic resizing
- This thing was repeated a few minutes earlier after that I closed an app and therefore its launcher (I was trying to close, most launchers possible,  to maximize icons size)

**Comment by psifidotos** [comment]
> Hidden and flickering dock

when this happen?

**Comment by psifidotos** [comment]
> I can see only very partial icons

do you have screenshot?

**Comment by JenaPlinsky** [comment]
I had the problem after only 14 seconds from inception
and there was no particular event
```
Ivan@UltrabookSamsung:~> latte-dock --debug
qt5ct: using qt5ct plugin
[debug **20:15:55**.311311] - package is valid true
............
............
............
[debug **20:16:09**.552552] - Something went wrong, received HTTP status code 403
```

now it is restarted
```
klauncher: Exiting on signal 15
[debug 21:07:24.648648] - Step 3 - found:40
[debug 21:08:05.418418] - Step 3 - found:40
[debug 21:09:13.936936] - Step 3 - found:40
```
![screenshot_20170131_210809](https://cloud.githubusercontent.com/assets/24593950/22482549/beb7a286-e7f9-11e6-8dd7-a6dda5fd95f8.png)

**Comment by JenaPlinsky** [comment]
and here again
```
[debug` 21:09:13.936936] - Step 3 - found:40
Creating the cache for:  "/home/Ivan"
Already in database?  true
      First update :  QDateTime(2017-01-30 01:02:41.000 CET Qt::TimeSpec(LocalTime))
       Last update :  QDateTime(2017-01-31 18:30:58.000 CET Qt::TimeSpec(LocalTime))
After the adjustment
     Current score :  7.8154
      First update :  QDateTime(2017-01-30 01:02:41.000 CET Qt::TimeSpec(LocalTime))
       Last update :  QDateTime(2017-01-31 18:30:58.000 CET Qt::TimeSpec(LocalTime))
Interval length is  0
Interval length is  0
         New score :  9.8154
ResourceScoreUpdated: "afaa6c8d-ff8b-49ab-b357-b2d2b48a4009" "org.kde.dolphin" "/home/Ivan"
[debug 21:16:14.300300] - klauncher not running... launching kdeinit
kdeinit5: Shutting down running client.
qt5ct: using qt5ct plugin
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
klauncher: Exiting on signal 15
"INSERT INTO ResourceInfo(   targettedResource, title, autoTitle, mimetype, autoMimetype) VALUES (  :targettedResource, '' , 1 , '' , 1 )"
QSqlError("5", "Impossibile ottenere la riga", "database is locked")
"INSERT INTO ResourceInfo(   targettedResource, title, autoTitle, mimetype, autoMimetype) VALUES (  :targettedResource, '' , 1 , '' , 1 )"
QSqlError("5", "Impossibile ottenere la riga", "database is locked")
Creating the cache for:  "/home/Ivan/Immagini/Screenshot_20170131_210809.png"
Already in database?  false
      First update :  QDateTime(2017-01-31 21:20:56.000 CET Qt::TimeSpec(LocalTime))
       Last update :  QDateTime(2017-01-31 21:20:56.000 CET Qt::TimeSpec(LocalTime))
After the adjustment
     Current score :  0
      First update :  QDateTime(2017-01-31 21:20:56.146 CET Qt::TimeSpec(LocalTime))
       Last update :  QDateTime(2017-01-31 21:20:56.000 CET Qt::TimeSpec(LocalTime))
         New score :  0
ResourceScoreUpdated: "afaa6c8d-ff8b-49ab-b357-b2d2b48a4009" "gimp" "/home/Ivan/Immagini/Screenshot_20170131_210809.png"
[warning 21:21:06.411411] - file:///usr/share/plasma/plasmoids/org.kde.latte.plasmoid/contents/ui/TaskDelegate.qml:213:9: QML Item: Binding loop detected for property "basicScalingHeight"
[debug 21:21:06.977977] - Step 3 - found:40
```

**Comment by JenaPlinsky** [comment]
It happens to you sometimes?

**Comment by psifidotos** [comment]
no, it doesnt...

can you send your settings? fill mode, zoom factor, maximum length and a screenshot how your dock looks without the problem...

**Comment by JenaPlinsky** [comment]
[lattedock-appletsrc.txt](https://github.com/psifidotos/Latte-Dock/files/743290/lattedock-appletsrc.txt)
[lattedockrc.txt](https://github.com/psifidotos/Latte-Dock/files/743291/lattedockrc.txt)
![screenshot_20170131_225413](https://cloud.githubusercontent.com/assets/24593950/22486304/3d28cf28-e809-11e6-8993-26a362700b5e.png)
![screenshot_20170131_225622](https://cloud.githubusercontent.com/assets/24593950/22486310/41d7fabc-e809-11e6-88b3-136c25256b53.png)
![screenshot_20170131_225648](https://cloud.githubusercontent.com/assets/24593950/22486313/49b09c12-e809-11e6-91c9-cbf0f7119515.png)
![screenshot_20170131_225947](https://cloud.githubusercontent.com/assets/24593950/22486315/4d3f3cd0-e809-11e6-9c64-44196565d870.png)

**Comment by JenaPlinsky** [comment]
Well, I'm ready and waiting for strange behavior, with advanced debug mode that you have just implemented

**Comment by psifidotos** [comment]
nice!! let's see...

**Comment by JenaPlinsky** [comment]
Well, the problem occurs (occasionally) when I close the tasks
[File di testo.txt](https://github.com/psifidotos/Latte-Dock/files/750630/File.di.testo.txt)
![screenshot_20170203_145528](https://cloud.githubusercontent.com/assets/24593950/22594153/2451446a-ea22-11e6-80bb-91be21b56418.png)

**Comment by JenaPlinsky** [comment]
and now re occurs again

hear before the issue 
![screenshot_20170203_151254](https://cloud.githubusercontent.com/assets/24593950/22594557/cdab5e00-ea23-11e6-805d-53b66f576a35.png)

and now after closing kcalc
[File di testo.txt](https://github.com/psifidotos/Latte-Dock/files/750659/File.di.testo.txt)

**Comment by JenaPlinsky** [comment]
I just upgraded from master
and I get the same previous result

konsole output: [File di testo.txt](https://github.com/psifidotos/Latte-Dock/files/752039/File.di.testo.txt)

![screenshot_20170204_063545](https://cloud.githubusercontent.com/assets/24593950/22616037/27a7c5fc-eaa5-11e6-9f77-0b7027b83d77.png)
flickering frame by frame ... mask value change
![spectacle v23870](https://cloud.githubusercontent.com/assets/24593950/22616110/884722e4-eaa6-11e6-96d0-552256b81231.png)
flickering frame by frame ... mask value change
![spectacle m23870](https://cloud.githubusercontent.com/assets/24593950/22616090/2c72e598-eaa6-11e6-993d-96fc53a55548.png)

**Comment by psifidotos** [comment]
@JenaPlinsky can you please check branch fix103 ?

**Comment by JenaPlinsky** [comment]
Amazing !!!!
there is a good chance that the problem has been fixed. I could not replicate it.
Partially also fixed the #128
